components/feed.jsx

```diff
@@ -0,0 +1,77 @@
+import styles from '../styles/Feed.module.css';
+import { useState, useEffect, useContext } from 'react';
+import axios from 'axios';
+import Carousel from 'react-elastic-carousel';
+import Image from 'next/image'
+import logo from '../public/Gamut_logo_small.png'
+import firebase from '../firebase.js'
+import { getAuth} from "firebase/auth";
+import NavBar from './navbar/Nav.jsx';
+import MainContext from "../context/MainContext";
+
+export default function Feed() {
+  const { exploreData } = useContext(MainContext);
+  const [filteredPosts, setFilteredPosts] = useState([]);
+  const [votes, setVotes] = useState(0);
+  const auth = getAuth();
+  let posts = filteredPosts.length === 0 ? exploreData : filteredPosts
+
+  const checkLoginStatus = () => {
+    if(auth.currentUser !== null) {
+      axios.get('http://localhost:3000/getPosts', {params: {email: auth.currentUser.email}})
+            .then((res) => {
+              setFilteredPosts(res.data)
+          }
+        )
+            .catch(err => console.log(err))
+    }
+  };
+
+  useEffect(()=>{
+    checkLoginStatus();
+  })
+
+  const handleSwipe = (index) => {
+    if(auth.currentUser !== null){
+      axios
+        .post(`http://localhost:3000/updateInteraction`, {email: auth.currentUser.email, postId: posts[index].id, interaction: votes})
```

Comment:
This would be something like

```
axios.post('/api/updateInteraction', {
    "email": auth.currentUser.email,
    "postId": posts[index].id,
    "interaction": votes
});
```